ceph_deploy/hosts/centos/install.py

```diff
@@ -4,6 +4,23 @@
 from ceph_deploy.util.paths import gpg
 
 
+def map_components(components):
```

Comment:
this is almost the same in the other distros, except for the packages that are split differently.

Would it be better if something does the work of intersecting the lists to determine the packages needed? It seems that
they all follow the same pattern, except for the change of items that pull in the ceph package. So something like:

    packages = set()
    for c in ceph_components:
        if c in components:
            components.remove(c)
            packages.append('ceph')

    for c in components:
        packages.append(c)

    return list(packages)

Would allow us to just define what the `ceph_components` are (mind you I didn't put much thought in naming) and the helper would get the same results.